### ISSUE TITLE
[Keyboard Navigation – Howdyai]: “Skip to Main content” link and "Back to Top" link is not defined for the page.

### ISSUE THREAD

**Issue by v-gasikk** [howdyai/howdy-website#2]
**User Experience:**
Users who depend on Keyboard are getting affected, if “Skip to Main content” and "Back to Top" link is not defined for the page.
 
**Test Environment:**
OS: Win 10 Version 1809 (OS Build 17763.379
URL: https://howdy.ai/
Browser: Edge
Tool Used: Developer tool bar (F 12)
 
**Repro Steps:**
Step 1: Open the above URL in Edge browser.
Step 2: Navigate to the page using keyboard tab key.
Step 3: Verify whether “Skip to main Content” and "Back to Top" link is defined for the page or not.
 
**Actual Result:**
“Skip to Main content” and "Back to Top"  links are not defined in the page.
 
**Expected Result:**
“Skip to Main content” and "Back to Top" links should be defined in the page.
User should able to Skip the header section that's why we need "Skip to main content" link.
 
**Note**: Same issue repros for all the pages in howdyai application.
 
**MAS Reference Link:**
[](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={20bce2aa-9952-4a10-af79-3376bcc8766d}) 
[MAS2.4.1_Back to top.zip](https://github.com/howdyai/howdy-website/files/2971106/MAS2.4.1_Back.to.top.zip)

**Comment by benbrown** [comment]
Back to top and skip to main links added to all pages on the site!

**Comment by v-gasikk** [comment]
Verified. Issue is Fixed. Hence Closing the bug
